src/app/domain/sharing/index.ts

```diff
@@ -0,0 +1,3 @@
+import Minilog from 'cozy-minilog'
+
+export const sharingLogger = Minilog('🗃️ SharingService')
```

Comment:
Do we really want to have a file just to make the economy of one variable declaration?

Reply:
multiple files use it and there's not really a single file more important than the others so in any case I would put it in a file apart